pyproject.toml

```
@@ -20,7 +20,7 @@ classifiers = [
 ]
 dependencies = [
     "numpy>=1.19.0",
-    "pandas<2.0.0" ,
+    "pandas>2.0.0" ,
```

```suggestion
    "pandas>=2.0.0" ,
```

Fixes #1, make sure requirements.txt is updated too